src/main/webapp/app/shared/util/core-drive-annotation-submission/core-drive-annotation-submission.ts

```diff
@@ -0,0 +1,136 @@
+import _ from 'lodash';
+import { Drug, Gene, Mutation, Review, Treatment, Tumor, Vus, DrugCollection } from '../../model/firebase/firebase.model';
+
+export function getGeneData(geneData: Gene, excludeComments: boolean, onlyReviewedContent: boolean, drugList: DrugCollection): true | Gene {
```

Comment:
Should we have all functions exposed and add few unit tests for each? Or you think the `getGeneData` and `getVUSData` covered them pretty well?

Reply:
My preference is to just test the public functions. If you start adding test every single function it become more difficult to refactor since you'll break a lot more of tests just because you refactored not because you wrote bugs.